src/hooks/useWalletCloudBackup.js

```diff
@@ -26,6 +27,27 @@ export default function useWalletCloudBackup() {
       password,
       walletId,
     }) => {
+      const isAvailable = await isCloudBackupAvailable();
+      if (!isAvailable) {
+        Alert.alert(
+          'iCloud Error',
```

Comment:
instead of Error, maybe something like "iCloud Not Enabled" or "Enable iCloud" ? @christianbaroni